CalendarApp/View Controllers/ComposeNoteViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  ComposeNoteViewController.swift
+//  CalendarApp
+//
+//  Created by Angelina Zhang on 8/2/22.
+//
+
+import Foundation
+
+protocol ComposeNoteDelegate: NSObject {
+    func didTapBack(note: Note, index: Int?);
+    func didUpdateNote(note: Note, index: Int);
+    func didDelete(noteID: UUID);
+    func didCreateNewNote(note: Note);
+}
+
+class ComposeNoteViewController : UIViewController, UITextViewDelegate, UITextFieldDelegate {
+    public var isNewNote = false
+    public var noteDidChange = false
+    public var note = Note()
```

Comment:
Ah. I see why you wanted to have the defaults... alternately you could wait until you have the necessary input to create this.